docs/go/testing.md

```diff
@@ -245,3 +247,14 @@ func (s *UnitTestSuite) Test_ProgressWorkflow() {
 	s.Equal(value, 100)
 }
 ```
+
+## Debugging
+
+In addition to writing unit and integration tests, debugging your Workflows is also a very valuable testing tool.
+You can debug your Workflow code using a debugger provided by your favorite IDE.
+
+Note that when debugging your Workflow code, the Temporal Go SDK includes deadlock detection which fails a Workflow Task in case the code blocks over a second without relinquishing execution control.
```

Comment:
A couple of things here:
1. Instead of "Workflow code" lets try to use Workflow Definition when referring to static Workflow code.
2. A Workflow Task is something very specific and I don't think an IDE debugging tool knows anything about a Workflow Task - it sounds like we are referring to a Workflow "awaitable" ? - like an Activity or something? 

```suggestion
The Temporal Go SDK includes deadlock detection which fails a Workflow awaitable (Activity Execution, Signal, Query) in case the code blocks over a second without relinquishing execution control.
```

Reply:
idk what is am "awaitable" . the code docs say "workflow task"